src/app/components/pop-up/pop-up.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-pop-up',
+  templateUrl: './pop-up.component.html',
+  styleUrls: ['./pop-up.component.scss']
+})
+export class PopUpComponent implements OnInit {
+
+  @Input() popUpName: string;
+  @Input() title: string;
+  @Input() open: boolean;
+
+  @Output() closeAction = new EventEmitter;
+
+  constructor() {}
+
+  ngOnInit(): void {}
+
+  close() {
+    this.open = false;
+    this.closeAction.emit(false);
```

Comment:
if you close why sent false on event >

Reply:
because if i don't send false, the parent doesn't change close state and doesn't reopen the pop-up

Reply:
ok, I understood that so i suggest more descriptive name